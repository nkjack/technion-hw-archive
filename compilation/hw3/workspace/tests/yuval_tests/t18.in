void main(){
  int x;
}
void checkIf(){
  bool x;
  if(x){
    print("ok");
    if (false) printi(5);
  }
  if () printi(5);
  else {
    int t;
  }
}