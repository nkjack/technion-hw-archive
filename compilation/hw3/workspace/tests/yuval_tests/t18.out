---end scope---
main - 0 preconditions
x INT 0
---end scope---
---end scope---
---end scope---
line 10: syntax error
